resources/todomvc/architecture-examples/lit/src/lib/todo-item.ts

```diff
@@ -131,18 +132,37 @@ export class TodoItem extends LitElement {
     @property({ type: Boolean })
         completed = false;
 
+    @property({ type: Number })
+        index = 0;
+
     @state()
         isEditing: boolean = false;
 
+    override connectedCallback() {
+        super.connectedCallback();
+        if (!EXTRA_CSS_TO_ADOPT)
+            return;
+        const styleSheetIndex = this.index % EXTRA_CSS_TO_ADOPT.length;
+        const styleSheetToAdopt = EXTRA_CSS_TO_ADOPT[styleSheetIndex];
+        this.shadowRoot?.adoptedStyleSheets.push(styleSheetToAdopt);
```

Comment:
should we check if a stylesheet exists at this index, before pushing?

Reply:
Oops I forgot the if. Done.